src/modules/__tests__/modules.spec.js

```diff
@@ -16,8 +16,8 @@ test('SceneModule', () => {
   modules.scene = new WHS.SceneModule();
 });
 
-test('CameraModule', () => {
-  modules.camera = new WHS.CameraModule();
+test('DefineModule', () => {
```

Comment:
when we modify classes, I think we should try to add assertions.
Maybe just check that data was populated with the passed in param (PerspectiveCamera), with the passed name?